### PR TITLE
feat: add counter for number of link created

### PR DESCRIPTION
**Pull request by ImOverlord**
Prom Counter for number of links created

### CODE REVIEW

File: server/metrics/response-duration.middleware.ts [hunk 1]
```diff
@@ -9,6 +9,13 @@ const responseDurationHistogram: Histogram = new Histogram({
   registers: [register]
 });
 
+const linkCounter = new Counter({
+  name: 'link_counter',
+  help: 'Number of link created',
+  labelNames: ['links'],
```
Comment (Tchoupinax) [inline server/metrics/response-duration.middleware.ts]: Why did you put that, I don't see where you provide it

**Comment by Tchoupinax** [comment]
Could your please trigger the /metrics route and copy/paste the metric rows you just add?

**Comment by ImOverlord** [comment]
> Could your please trigger the /metrics route and copy/paste the metric rows you just add?
```
# HELP link_counter Number of link created
# TYPE link_counter counter
link_counter 4
```

File: server/metrics/response-duration.middleware.ts (hunk 1 above)
Comment (ImOverlord) [reply server/metrics/response-duration.middleware.ts]: removed

**Comment by Tchoupinax** [comment]
LGTM, thank you for your job!

**Pull request merged**